coral-component-table/src/scripts/Table.js

```diff
@@ -2168,7 +2168,31 @@ class Table extends BaseComponent(HTMLTableElement) {
   _onSelectPreviousItem() {
     this._selectSiblingItem(false);
   }
-  
+
+  /**
+  * Call the layout method of table component
+  *
+  * @param {Boolean} forced
+  * If true call the layout method immediately, else wait for timeout
+  */
+  resetLayout(forced){
+    if(forced === true){
```

Comment:
Please transform the param with `forced = commons.boolean(forced);` 
Also can you pls add space after `if` statement ?